ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/claimantresponse/CourtDeterminationMapper.java

```diff
@@ -37,7 +37,8 @@ public CourtDetermination from(CCDCourtDetermination ccdCourtDetermination) {
         CourtDetermination.CourtDeterminationBuilder builder = CourtDetermination.builder()
             .courtDecision(paymentIntentionMapper.from(ccdCourtDetermination.getCourtDecision()))
             .disposableIncome(ccdCourtDetermination.getDisposableIncome())
-            .courtPaymentIntention(paymentIntentionMapper.from(ccdCourtDetermination.getCourtIntention()));
+            .courtPaymentIntention(paymentIntentionMapper.from(ccdCourtDetermination.getCourtIntention()))
+            .decisionType(ccdCourtDetermination.getDecisionType());
         if (ccdCourtDetermination.getRejectionReason() != null) {
```

Comment:
I am trying to avoid having element "rejectionReason" : null in the courtDetermination by default because AFAIK this is only used in interlocutory judgement.

Reply:
as we are not doing any operation on rejection reason, no need null check and based on our current configuration, Object mapper ignores null fields.